clang/include/clang/Driver/Options.td

```diff
@@ -1980,9 +1980,11 @@ within a command line are combined with spaces; spaces and backslashes within an
 argument are escaped with backslashes. This format differs from the format of
 the equivalent section produced by GCC with the -frecord-gcc-switches flag.
 This option is currently only supported on ELF targets.}]>,
-  Group<f_clang_Group>;
+  Group<f_clang_Group>,
```

Comment:
Do you know whether flang's flow is affected by `f_clang_Group`?

Reply:
Could you clarify what you mean by "flang's flow"? 

I don't know exactly what `f_clang_Group` does. I haven't been able to determine what effect it has on things like help messages. For instance `-frecord-command-line` does not appear in either `clang --help`, or `clang --help-hidden`.

Reply:
@kiranchandramohan, gentle ping

Reply:
I mean't whether `f_clang_Group` is processed by flang or not. I see that this is used by some other commands as well. So it is alrite.

Reply:
I think f_clang_Group is a legacy group for Clang options not in GCC. f_clang_Group has never gain significant adoption and it's used by any code.

New -f options can just use f_Group.

Reply:
I have changed the group to `f_Group` since that does not suggest that the options are exclusive to clang.